tests/db/test_bigchain_api.py

```diff
@@ -1269,22 +1130,22 @@ def test_get_spent_single_tx_multiple_outputs(self, b, user_sk, user_vk):
         for input_tx in owned_inputs_user1:
             assert b.get_spent(input_tx.txid, input_tx.cid) is None
 
-        # select inputs to use
-        inputs = sum([tx.to_inputs() for tx in transactions[:2]], [])
-
-        # create a transaction and block
-        tx = Transaction.transfer(inputs, len(inputs) * [[user2_vk]])
-        tx = tx.sign([user_sk])
-        block = b.create_block([tx])
+        # transfer the first 2 inputs
+        tx_transfer = Transaction.transfer(tx_create.to_inputs()[:2],
```

Comment:
Formerly this test spent inputs from different transactions. Any reason why we got rid of this?